configs/data/im2im/numpy_dataloader_train.yaml

```diff
@@ -0,0 +1,79 @@
+_aux:
+  patch_shape:
+  _scales_dict:
+    - - ${target_col}
+      - [1]
+    - - ${source_col}
+      - [1]
+
+train_dataloaders:
+  _target_: cyto_dl.datamodules.array.make_array_dataloader
+  data:
```

Comment:
is this needed?

Reply:
Do you mean the `data` key? or the full .yaml?

Reply:
I meant data, but realized later that this is where the numpy array is passed in